### DISCUSSION TITLE
IP addresses of LAN, WAN etc.

### DISCUSSION THREAD

**Discussion by Clueless2015** [travisghansen/hass-pfsense#103] [Q&A]
Hi,

Does anyone know if there are entities created to show the interface IP addresses?  Specifically i'm looking to monitor my Internet IPs from a couple of interfaces and can't seem to find this piece of data within the integration.

Many thanks!

**Comment by RJ1002** [comment]
yes there is ip addresses. in each device_tracker has attributes and there one named last_known_ip

**Reply by Clueless2015** [reply]
Thanks, unfortunately i don't have last_known_ip or related in entities.  Nor anything IP related where I would expect it within the _interface_ entities grouping.
